WTWSMS/events/ze_lord_spiritual_events.txt

```diff
@@ -29,7 +29,6 @@ character_event = {
 			independent = yes
 			higher_tier_than = DUKE
 		}
-		proselyte_religion_trigger = yes
```

Comment:
Don't be so dramatic.

Removing that one line does not somehow fundamentally change your LI crap. Everything else is still completely play. This effects ONE Decision in ONE file, changing ONE event, and it makes that ONE event more dynamic, allowing for greater variety of possible conversions, while maintaining that the AI of a non-prolestyze religion isn't that likely to do it. Besides, the code itself says thatm embers of a prolestyzing religion are more likely to convert their subjects. It has nothing to do with missionary work.